composer.json

```diff
@@ -4,11 +4,6 @@
     "homepage": "https://github.com/shopgate/connect-integration-sdk",
```

Comment:
dont forget to change the name due to repo change! + dont forget to change the composer name as well.
Also remove the shopgate.php file from autoloading (and delete it), we don't need it in the next implementations.

Reply:
good, but u still forgot the exclude file list in the deploy folder, gitignore and you did not change this line of code that I was highlighting